packages/manager/src/components/PrimaryNav/PrimaryNav.tsx

```diff
@@ -288,6 +291,13 @@ export const PrimaryNav = (props: PrimaryNavProps) => {
           href: '/longview',
           icon: <Longview />,
         },
+        {
+          display: 'Monitor',
+          hide: !showCloudPulse,
+          href: '/monitor/cloudpulse',
+          icon: <CloudPulse />,
+          isBeta: false,
```

Comment:
```suggestion
          isBeta: flags.aclp?.beta,
```

Using the feature flag here will allow us to toggle the visibility of the beta chip via feature flag, so a future code change will not be needed. 
![Screenshot 2024-05-15 at 9 35 48 AM](https://github.com/linode/manager/assets/114685994/52cb91c1-8363-494d-b836-306205065d85)

Reply:
Yeah, pushing the changes now. Thank you!